### DISCUSSION TITLE
CyberColorPicker inital value

### DISCUSSION THREAD

**Discussion by rrirower** [Taiizor/ReaLTaiizor#195] [Q&A] [answered]
Is there any way to set the initial color value for the CyberColorPicker and have it reflected in the the color wheel?  I've set the 'SelectedColor' value, but, the color wheel still shows the color in the center of the color wheel.

**Comment by Taiizor** [comment, marked as answer]
This issue will be resolved in the next release.

![image](https://github.com/user-attachments/assets/25dbbef2-3714-4a84-98c1-fecafdade0db)

![image](https://github.com/user-attachments/assets/5f2b5831-adea-48cc-95cd-8d53fda6f3ae)

**Reply by rrirower** [reply]
Thank you.
